iOS internship/ForecastTransport.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  ForecastTransport.swift
+//  iOS internship
+//
+//  Created by pavels.vetlugins on 26/09/2018.
+//  Copyright © 2018 pavels.vetlugins. All rights reserved.
+//
+
+import Foundation
+import Freddy
+
+
+class ForecastTransport{
+    
+    var delegate: FetchForecastDelegate?
+    
+    let host = "http://dataservice.accuweather.com"
```

Comment:
This should all be `private` if it's not required outside this class. Always state encapsulation explicitly, everything that is not required to be `public` or `internal` should be `private`.